### PR TITLE
Pytest

### PR DESCRIPTION
**Pull request by jcacosta89**
That file should not be check.  I copy and paste the whole script but it is reading the whole script as new. **[hhnk_threedi_tools/utils/notebooks/03_nabewerking_klimaatsommen_prep.ipynb]** 

I updated this file to include **excel files and json file** [tests/data/model_test/02_schematisation/.gitignore]

I deleted this two files from that location to place them correctly in the model schematitation. They are deleted **tests/data/model_test/model_settings_default.xlsx - tests/data/model_test/model_settings_default.xlsx**

### CODE REVIEW

File: tests/data/model_test/02_schematisation/model_settings.xlsx
Comment (wvangerwen) [inline tests/data/model_test/02_schematisation/model_settings.xlsx:1]: Make sure this is properly merged with PR #51

File: hhnk_threedi_tools/core/checks/sqlite/sqlite_main.py [hunk 1]
```diff
@@ -494,82 +494,55 @@ def create_grid_from_sqlite(self, sqlite_path, dem_path, output_folder):
...
-                count = cross_sec_id.count(_)
-                counter.append(count)
-
-            gdf_data = gpd.GeoDataFrame({"cross_loc_id_left":cross_location_id, 'count': counter})
-            gdf_data =  gdf_data.drop_duplicates(['cross_loc_id_left','count'],keep = 'last')
-            gdf_data = gdf_data[gdf_data['count']> 1 ]
-            list_id = gdf_data.cross_loc_id_left.values.tolist()
-            cross_section_warning = cross_section_point[cross_section_point['cross_loc_id'].isin(list_id)]
-            return cross_section_warning 
+            # Een foutmelding geven als de volgen de conditie.
```
Comment (wvangerwen) [inline hhnk_threedi_tools/core/checks/sqlite/sqlite_main.py:507]: Printing a full table isnt great on the main

File: tests/test_sqlite.py
Comment (wvangerwen) [inline tests/test_sqlite.py:1]: tests look alright, please look into the naming

**Review by wvangerwen — CHANGES_REQUESTED** [review]
My qgis crashes alot when i have this branch activated. Please look into that.
Simply open MI:
-> sqlite checks
-> run all or just generate grid
-> if it works in one go delete layers and try again. For me it crashes (tried 5 restarts. Reverted back to main and all worked again.)

File: hhnk_threedi_tools/utils/notebooks/03_nabewerking_klimaatsommen_prep.ipynb
[the remark this answers is not in the document]
Comment (jcacosta89) [reply hhnk_threedi_tools/utils/notebooks/03_nabewerking_klimaatsommen_prep.ipynb:1]: deleted

File: tests/data/model_test/02_schematisation/.gitignore
[the remark this answers is not in the document]
Comment (jcacosta89) [reply tests/data/model_test/02_schematisation/.gitignore:1]: delete from gitignore: model settings, model setting default and json file

File: tests/data/model_test/02_schematisation/model_settings.xlsx
Comment (jcacosta89) [reply tests/data/model_test/02_schematisation/model_settings.xlsx:1]: Merge done

File: hhnk_threedi_tools/core/checks/sqlite/sqlite_main.py (hunk 1 above)
Comment (jcacosta89) [reply hhnk_threedi_tools/core/checks/sqlite/sqlite_main.py:507]: deleted printing lines  from both functions:  run_cross_section_vertex and run_cross_section

**Review by wvangerwen — APPROVED** [review]

**Pull request merged**